core/io/file_access_zip.h

```diff
@@ -69,7 +69,7 @@ class ZipArchive : public PackSource {
 
 	bool file_exists(String p_name) const;
 
-	virtual bool try_open_pack(const String &p_path, bool p_replace_files);
+	virtual bool try_open_pack(const String &p_path, bool p_replace_files, size_t p_offset);
```

Comment:
We also need to modify function signature here because it also implements the pure virtual try_open_pack() function.

Reply:
Loading ZIP package is not within the scope of this PR (Install-time packs will use the format of .PCK package since they are already zipped inside of .apk when downloaded). Adding such feature will also require low level modification to Zlib. [here](https://github.com/godotengine/godot/blob/00949f0c5fcc6a4f8382a4a97d5591fd9ec380f8/core/io/file_access_zip.cpp#L172) and [here](https://github.com/godotengine/godot/blob/24e1039eb6fe32115e8d1a62a84965e9be19a2ed/thirdparty/minizip/unzip.h#L184)